src/nls/it/strings.js

```diff
@@ -648,7 +648,104 @@ define({
+    "DESCRIPTION_LANGUAGE_FILE_NAMES"                : "Ulteriori mappature da nome di file a nome di lingua",
+    "DESCRIPTION_LINTING_ENABLED"                    : "Vero abilità Code Inspection",
+    "DESCRIPTION_ASYNC_TIMEOUT"                      : "Il tempo in millisecondi dopo la pausa del linter asincrono",
+    "DESCRIPTION_LINTING_PREFER"                     : "Vettore di linter da eseguire prima",
+    "DESCRIPTION_LIVE_DEV_MULTIBROWSER"              : "Vero abilita Esperimento Anteprima Live",
+    "DESCRIPTION_USE_PREFERED_ONLY"                  : "Vero avvia fornitori solo da linting.prefer",
+    "DESCRIPTION_MAX_CODE_HINTS"                     : "Massimo numero di suggerimenti visualizzati alla volta",
+    "DESCRIPTION_PATH"                               : "Impostazioni di percorso specifiche",
+    "DESCRIPTION_PROXY"                              : "L'URL del server proxy utilizzato per l'installazione ampliamento",
+    "DESCRIPTION_SCROLL_PAST_END"                    : "Vero per attivare lo scorrimento oltre la fine del documento",
+    "DESCRIPTION_SHOW_CODE_HINTS"                    : "Falso per disattivare tutti i suggerimenti sul codice",
+    "DESCRIPTION_SHOW_CURSOR_WHEN_SELECTING"         : "Mantiene il cursore lampeggiante visibile quando si dispone di una selezione di testo",
+    "DESCRIPTION_SHOW_LINE_NUMBERS"                  : "Vero per mostrare i numeri di riga in un " grondaia " a sinistra del codice",
+    "DESCRIPTION_SMART_INDENT"                       : "Automaticamente trattino quando si crea un nuovo blocco",
+    "DESCRIPTION_SOFT_TABS"                          : "False per disattivare il comportamento soft tab",
+    "DESCRIPTION_SORT_DIRECTORIES_FIRST"             : "Vero per ordinare prima le cartelle nella struttura del progetto",
+    "DESCRIPTION_SPACE_UNITS"                        : "Numero di spazi da utilizzare per il rientro dello spazio",
+    "DESCRIPTION_STATIC_SERVER_PORT"                 : "Numero di porta che il built-in server dovrebbe usare per Anteprima Live",
+    "DESCRIPTION_STYLE_ACTIVE_LINE"                  : "Vero per evidenziare il colore della linea di fondo si trova il cursore",
+    "DESCRIPTION_TAB_SIZE"                           : "Numero di spazi da visualizzare per le tabulazioni",
+    "DESCRIPTION_USE_TAB_CHAR"                       : "Vero utilizzare le tabulazioni al posto degli spazi",
+    "DESCRIPTION_UPPERCASE_COLORS"                   : "Vero per generare caratteri maiuscoli nei colori hex in Inline Color Editor",
+    "DESCRIPTION_WORD_WRAP"                          : "Riga ad ogni ritorno la larghezza viewport",
+    "DESCRIPTION_DETECTED_EXCLUSIONS"                : "Un elenco dei file che sono stati rilevati per causare a Tern di correre fuori controllo",
+    "DESCRIPTION_INFERENCE_TIMEOUT"                  : "La quantità di tempo dopo il quale Tern scadrà quando si cerca di capire i file",
```

Comment:
**`Il tempo in millisecondi`**` dopo il quale Tern `**`viene interrotto mentre cerca di leggere un file`**